egs/swbd/s5c/RESULTS

```diff
@@ -142,7 +142,3 @@ exit 0
 %WER 14.6 | 1831 21395 | 87.1 9.2 3.6 1.8 14.6 51.8 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_tg_it4/score_17_0.0/eval2000.ctm.swbd.filt.sys
 # fisher 4gram rescoring,
 %WER 13.2 | 1831 21395 | 88.3 8.2 3.4 1.5 13.2 49.2 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_fsh_fg_it4/score_19_0.0/eval2000.ctm.swbd.filt.sys
-
-# nnet3 LSTM recipe local/nnet3/run_lstm.sh (30.09.2015), swbd subset
-%WER 16.8 | 1831 21395 | 84.8 10.4 4.7 1.6 16.8 54.3 | exp/nnet3/lstm_ld5/decode_eval2000_sw1_tg/score_10_0.0/eval2000.ctm.swbd.filt.sys
-%WER 15.7 | 1831 21395 | 85.6 9.6 4.8 1.3 15.7 52.8 | exp/nnet3/lstm_ld5/decode_eval2000_sw1_fsh_fg/score_12_0.0/eval2000.ctm.swbd.filt.sys
```

Comment:
Thanks- perhaps you could create a pull request for this, including the
scripts?
Dan


On Wed, Oct 14, 2015 at 12:59 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/swbd/s5c/RESULTS
> <https://github.com/kaldi-asr/kaldi/pull/249#discussion_r42020918>:
>
> > @@ -142,7 +142,3 @@ exit 0
> >  %WER 14.6 | 1831 21395 | 87.1 9.2 3.6 1.8 14.6 51.8 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_tg_it4/score_17_0.0/eval2000.ctm.swbd.filt.sys
> >  # fisher 4gram rescoring,
> >  %WER 13.2 | 1831 21395 | 88.3 8.2 3.4 1.5 13.2 49.2 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_fsh_fg_it4/score_19_0.0/eval2000.ctm.swbd.filt.sys
> > -
> > -# nnet3 LSTM recipe local/nnet3/run_lstm.sh (30.09.2015), swbd subset
> > -%WER 16.8 | 1831 21395 | 84.8 10.4 4.7 1.6 16.8 54.3 | exp/nnet3/lstm_ld5/decode_eval2000_sw1_tg/score_10_0.0/eval2000.ctm.swbd.filt.sys
> > -%WER 15.7 | 1831 21395 | 85.6 9.6 4.8 1.3 15.7 52.8 | exp/nnet3/lstm_ld5/decode_eval2000_sw1_fsh_fg/score_12_0.0/eval2000.ctm.swbd.filt.sys
>
> these are results with nnet3 LSTMs : local/nnet3/run_lstm.sh (8 epoch
> training on speed-perturbed and volume perturbed data)
>
> %WER 11.8 | 1831 21395 | 89.4 7.0 3.6 1.3 11.8 47.8 |
> exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_fsh_fg/score_13_0.0/eval2000_hires.ctm.swbd.filt.sys
> %WER 13.2 | 1831 21395 | 88.3 8.0 3.7 1.5 13.2 49.5 |
> exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
>
> # results with nnet3 tdnn: local/nnet3/run_tdnn.sh (11.10.2015) (2 epoch
> training on speed-perturbed and volume perturbed data)
> %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 |
> exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
> %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 |
> exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/249/files#r42020918>.
>